assets/js/components/leaflet/leaflet.css

```diff
@@ -110,11 +110,11 @@
 	width: 1px;
 	height: 1px;
 	}
+	
```

Comment:
Sarebbe meglio non modificare assets di plugin esterni, perché in caso di aggiornamento finiscono per essere sovrascritti

Reply:
La modifica al file CSS di leaflet è stata rimossa.